app/assets/javascripts/backbone/views/person_view.coffee

```diff
@@ -0,0 +1,14 @@
+# Allows user specific fields to be hidden / show based on checkbox
+class MS.Views.PersonView extends Backbone.View
+
+  events:
+    'click #person_has_system_access': 'showHideUserFields'
+
+  showHideUserFields: ->
+    if $('#person_has_system_access').is(':checked')
+      $('#user_fields').show()
+    else
+      $('#user_fields').hide()
```

Comment:
Pretty much all calls to `$` should be `@$` instead. It just ensures they stay within the scope of the View.